src/EnergyPlus/FluidProperties.cc

```diff
@@ -2759,8 +2752,6 @@ namespace FluidProperties {
 
         int LoTempIndex = FindArrayIndex(Temperature, this->PsTemps, this->PsLowTempIndex, this->PsHighTempIndex);
 
-        auto &df = state.dataFluidProps;
```

Comment:
Why undo this?

Reply:
The references are staying in place (I put them all back in) and only moving if there was a CppCheck reduce scope suggestion. I do want to see CI results though because the Mac results are troubling.

Reply:
Just to be clear, Mac results are not reliable right now.  There is something very odd going on that I'm trying to investigate.  Gort and Tik-Tok are doing fine though.